config/routes.rb

```diff
@@ -189,6 +189,11 @@
             param: :guid,
             only: [:show, :index]
 
+  # service_route_bindings
+  resources :service_route_bindings,
+    param: :guid,
```

Comment:
This is probably fine to keep since there's likely a show or update or delete in the future, but just so you know, this is the key for the resource (opposed to resources/list) endpoints which by convention is keyed off as `:id` in the params.